internal/pkg/selinux/policy/selinux/services/cri.cil

```diff
@@ -15,3 +15,30 @@
 (call pod_p (etcd_t))
 ; FIXME: insecure as anyone with access to the pod containerd may obtain this domain (executable in ephemeral)
```

Comment:
is the comment still valid?

Reply:
Yes. Ephemeral label actually comes later in the series, and even after then we need to think how to label kubelet/etcd executables and assign labels to static pods

Reply:
okay, let's keep what we have then, unless we figure we need to relabel it later